lxl-web/src/lib/utils/search.ts

```diff
@@ -31,6 +32,22 @@ import { getUriSlug } from '$lib/utils/http';
 // import { error } from '@sveltejs/kit';
 // import { env } from '$env/dynamic/public';
 
+export async function asLibraryResult(
```

Comment:
I kind of think `asLibraryResult` should be a local function of the api/my-pages/server file... I can't really see it being reused in some other case, so the export/import thing is kind of unnecessary?